data/mods/My_Sweet_Cataclysm/sweet_furniture.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "type": "furniture",
+    "id": "f_rubble_rockcandy",
+    "name": "pile of rock candy rubble",
+    "description": "Pile of rock candy shards.  Useless?",
+    "symbol": "^",
+    "color": "dark_gray",
+    "move_cost_mod": 6,
+    "max_volume": "750 L",
+    "required_str": -1,
+    "flags": [
+      "TRANSPARENT",
+      "UNSTABLE",
+      "ROUGH",
+      "PLACE_ITEM",
+      "MOUNTABLE",
+      "CONTAINER",
+      "SEALED",
+      "ALLOW_FIELD_EFFECT",
+      "SHORT",
+      "RUBBLE"
+    ],
+    "examine_action": "rubble"
+  },
+  {
+    "type": "furniture",
+    "id": "f_candy_boulder_small",
+    "name": "small rock candy boulder",
+    "description": "Blocking your path.  Should be easy to move.  It can be used as a primitive anvil.",
+    "symbol": "o",
+    "color": "dark_gray",
+    "move_cost_mod": 3,
+    "coverage": 30,
+    "required_str": 10,
+    "crafting_pseudo_item": "boulder_anvil",
+    "flags": [ "TRANSPARENT", "MINEABLE", "UNSTABLE", "MOUNTABLE", "TINY" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "smash!",
+      "sound_fail": "thump.",
+      "items": [ { "item": "candy3", "count": [ 1, 6 ] }, { "item": "marshmallow", "count": [ 0, 2 ] } ]
```

Comment:
It's an anvil but it has marshmallows inside? X)

Reply:
Maybe stuffs like pure unrefined sugar, colorants, flavour extracts etc..

Reply:
I wasn't sure how much incidental horror the candyverse would contain.  :) I'll work on some additions.  Would you want rock candy to be similar in material to glass?

Reply:
If you want, glass is a kind of rock after all.

Reply:
Let me know if this needs any changes :)